Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2227,6 +2227,41 @@ extension BrowserViewController: WKUIDelegate {
             self.tabManager.removeTab(tab)
         }
     }
+    
+    func webView(_ webView: WKWebView, shouldPreviewElement elementInfo: WKPreviewElementInfo) -> Bool {
+        guard let url = elementInfo.linkURL, !Storage.isIgnoredURL(url) else { return false }
+        return true
+    }
+    
+    func webView(_ webView: WKWebView, commitPreviewingViewController previewingViewController: UIViewController) {
+        guard let previewViewController = previewingViewController as? PreviewViewController,
+            let tab = tabManager.selectedTab,
```

Comment:
Don't need to unwrap `tab` or `webview` here, can just let it silently "fail"

`tabManager.selectedTab?.webView?.load(URLRequest(url: previewViewController.url))`

Reply:
Good point 💪 fixed.